### PR TITLE
Cleanup

### PR DESCRIPTION
**Pull request by paulSambolin**

### CODE REVIEW

File: sweet-skills-infrastructure/_full-stack.template [hunk 1]
```diff
@@ -71,7 +71,7 @@
         "BaseTemplateURL": {
             "Description": "S3 base URL for all the CloudFormation templates being used (without the file names)",
             "Type": "String",
-            "Default": "https://s3.amazonaws.com/honey-badger-cloud-formation/sweet-skills-infrastructure/"
```
Comment (ironmanwaring) [inline sweet-skills-infrastructure/_full-stack.template]: i think this needs to be reverted back

File: sweet-skills-infrastructure/web-routing.template [hunk 2]
```diff
@@ -82,7 +82,10 @@
             "sa-east-1":        { "s3DnsName": "s3-website-sa-east-1.amazonaws.com", "hostedZoneId" : "Z7KQH4QJS55SO" }
         }
     },
-    "Conditions": { },
+    "Conditions": {
+        "QuickStart": { "Fn::Equals" : [ { "Ref": "Environment" }, "personal" ] },
```
Comment (ironmanwaring) [inline sweet-skills-infrastructure/web-routing.template]: maybe something more descriptive here for what's happening?  notquickstart -> cloudfront, cloudfrontdistribution, securehosting, etc and quickstart -> nonsecurehosting, route53, etc

**Comment by ironmanwaring** [comment]
added two comments

File: sweet-skills-infrastructure/_full-stack.template (hunk 1 above)
Comment (paulSambolin) [inline sweet-skills-infrastructure/_full-stack.template]: back to /test/?

**Comment by ironmanwaring** [comment]
if it works, lgtm

**Comment by paulSambolin** [comment]
If it works it's good to me too

**Pull request merged**